test/suites/standard1/configuration.robot

```diff
@@ -70,3 +108,22 @@ Setup With Debug Log Level
     ${merged}=    Extend MicroShift Config    ${BAD_LOG_LEVEL}
     Upload MicroShift Config    ${merged}
     Restart MicroShift
+
+Setup With Unknown Audit Log Profile
+    [Documentation]    Set audit log profile to unknown value
+    ${merged}=    Extend MicroShift Config    ${BAD_AUDIT_PROFILE}
+    Upload MicroShift Config    ${merged}
+    Stop MicroShift
+    Start MicroShift
+
+Setup Known Audit Log Profile
+    [Documentation]    Setup audit
+    ${merged}=    Extend MicroShift Config    ${AUDIT_PROFILE}
+    Upload MicroShift Config    ${merged}
+    Restart MicroShift
```

Comment:
I'm wondering if we should refresh the ${CURSOR} before restarting so that we look at more specific log span? Otherwise it seems to use CURSOR set up at the beginning of this suite